### PR TITLE
Bos Taurus paragraph and other edits

### PR DESCRIPTION
**Pull request by gregorgorjanc**
Here is the Bos Taurus paragraph and a number of other edits - hopefully an improvement ;)

### CODE REVIEW

File: adding_species_manuscript.md [hunk 1]
```diff
@@ -334,16 +334,44 @@ We use species worked on during the hackathon to illustrate the process of choos
...
-        -   reasons for adding
-
-             - ag species, hence important for breeding and genetic simulations of applied problems, wild ??? extinct
-            
-            - Ne inferred to very recent times using a specific haplotype method (@MacLeod2013)
-            - decreasing Ne due to domestication and recent intense selection, this scenario is quite different to many other cases (stuff like GWAS is very limited due to long-range LD, but genomic prediction works really well withing populations, ... cite @Meuwissen2001, @MacLeod2014, https://royalsocietypublishing.org/doi/abs/10.1098/rspb.2016.0569)
-
-        -   can be used for methods development and inference
+    
+We have added Bos Taurus (cattle) species as an example of an agricultural
```
Comment (lauterbur) [inline adding_species_manuscript.md]: _Bos taurus_

File: adding_species_manuscript.md [hunk 2]
```diff
@@ -334,16 +334,44 @@ We use species worked on during the hackathon to illustrate the process of choos
...
+species. Agricultural species have generally experienced strong reduction in
+effective population size over time due to selection caused by domestication and
+recently selective breeding. Both of these processes have occurred over a
```
Comment (lauterbur) [inline adding_species_manuscript.md]: ~~recently~~ selective breeding

File: adding_species_manuscript.md [hunk 3]
```diff
@@ -334,16 +334,44 @@ We use species worked on during the hackathon to illustrate the process of choos
...
+@VanRaden2020). This boom inspired numerous simulation studies to study optimal
+uses of genomic data in selective breeding (@Meuwissen2001, @MacLeod2014, and
+others). Cattle is interesting for population genetic studies due to its
+decreasing effective population size. In the cosmopolitan dairy breed Holstein,
+the effective population size was estimated to be as low as ~90 around 1980
+(@MacLeod2013) and has since decreased even further due to intense selective
+breeding (e.g., @VanRaden2020, @Makanjouloa2020). This decrease is interesting
+because it is challenging the estimation of demography (@MacLeod2013),
+genome-wide association and prediction (@MacLeod2014), and inference of selection
+(@Hartfield2022). With respect to the parameters chosen in the stdpopsim
```
Comment (lauterbur) [inline adding_species_manuscript.md]: I think we can consolidate the introductory paragraph here: Suggest replacing lines 343-355 with "In cattle, high quality genome assemblies are now available for several breeds and population genomic analyses have become widely used to improve selective breeding and genomic prediction. Their small effective population size (~90 around 1980, and continuing to decline due to intense selective breeding) makes demographic and selection inference, as well as genome-wide association and prediction, challenging. For these reasons, it was useful to develop a standardized population genomic cattle model for stdpopsim."

Comment (lauterbur) [reply adding_species_manuscript.md]: (with appropriate citations of course)

Comment (lauterbur) [inline adding_species_manuscript.md]: New paragraph at "With respect to the parameters..."

**Comment by lauterbur** [comment]
Unless you want to change anything before I do, I will go ahead and merge these so I can do all-over edits.

**Comment by lauterbur** [comment]
I don't think Bos taurus needs any more cuts/major edits for now. We'll see how things flow with the rest once we have the conclusion.
Thanks for the edits elsewhere too, very helpful!

**Comment by gregorgorjanc** [comment]
@lauterbur yeah merge this in! Thanks! Sorry for dragging my feet!

One comment though - we need something similar for all other added species too, right?

**Comment by lauterbur** [comment]
I think that's overkill, one example for why a species might be of general interest, and how to pick parameter values, is probably enough since this is now a more general "how to do a popgen simulation" paper rather than a "uses of stdpopsim" paper

**Comment by gregorgorjanc** [comment]
OK, I was thinking the same, just wasn’t sure if the direction of travel.

With regards,
  Gregor

**Pull request merged**